### PR TITLE
Updated and additional language for Grade Levels

### PR DESCRIPTION
**Pull request by pixelwake**
Proposed language to help applicant choose which grade level to select when submitting an application.

### CODE REVIEW

File: pages/pay-grades.md [hunk 1]
```diff
@@ -2,14 +2,53 @@
 permalink: /pay-grades/
 title: Government pay grades explained
 ---
+### General Schedule – Grade Levels
+Federal employees are hired at a general schedule (GS) grade from 1 to 15. GS grades are based on the combination of a number of variables including education, background, accomplishments, experience, etc. Inside each grade is a series of 10 steps. Employees typically start at step one of whatever GS grade they are assigned.
```
Comment (fureigh) [inline pages/pay-grades.md]: Let's standardize to either "steps" or "levels" throughout.

**Comment by fureigh** [comment]
@rooneywp Thanks for this! Please take a look at the changes I made — admittedly out of order with standard pull request process, pardon — and advise as to whether it's better to standardize to 'steps' or 'levels' throughout.

**Comment by pixelwake** [comment]
@fureigh Sorry if I used the pull request incorrectly! This was my first one and just decided to give it shot.

Steps and Levels are referring to two separate things. (Grade being the GS 13 and step being the .1 in GS 13.1). I can understand the confusion so we should make sure that it's always "grade level" and not just "level".

**Comment by pixelwake** [comment]
I did some quick research to verify that we weren't falsely introducing "level" into the lexicon for GS pay and we aren't. OPM refers to grade level, "Because of serious difficulties in staffing certain occupations at GS grade levels in certain geographic areas".

I think as long as we don't level on it's own then we should be clear.

File: pages/pay-grades.md [hunk 2]
```diff
@@ -2,14 +2,57 @@
...
+- Grade 13 – $73,115 to $95,048
+- Grade 14 – $86,399 to $112,319
+- Grade 15 – $101,630 to $132,122
+
+**Those ranges are the base only; they do not include any adjustments for locality pay.** Here is a [GS Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2015/general-schedule-gs-salary-calculator/) if you would like to see how locality affects salary.
+
+| |Base (No Locality) |Washington DC Locality | San Francisco Locality |
+|---:|:---:|:---:|:---:|
+|**Grade 14**| $86,399|$107,325|$116,768|
+
```
Comment (pixelwake) [inline pages/pay-grades.md:49]: Added the table above to highlight the impact of locality.

File: pages/pay-grades.md [hunk 3]
```diff
@@ -2,14 +2,61 @@
...
+Locality pay is a geographic-based percentage rate that reflects pay levels for non-federal workers in certain geographic areas as determined by [surveys conducted by the U.S. Bureau of Labor Statistics](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/fact-sheets/tabs/bls-data/). There are currently [34 locality pay areas](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/fact-sheets/tabs/locality-pay-areas/), which cover the lower 48 states and Washington, D.C., plus Alaska, Hawaii, and the U.S. territories and possessions.
+
+Each GS grade level contains a series of 10 steps which also have an impact on compensation. A new GS employee is usually hired at step 1 of the applicable GS grade. However, in special circumstances, agencies may authorize a higher step rate for a newly-appointed federal employee based on a [special need of the agency or superior qualifications of the prospective employee](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/superior-qualifications-and-special-needs-pay-setting-authority/).
+
+### Joining 18F
```
Comment (pixelwake) [inline pages/pay-grades.md]: @fureigh Talking to the team, they proposed moving the Joining 18F & Selecting Grade level sections to the top of this page.

I could go either way, leaning more toward moving to the top. If we are going to be directing applicants to this section from the application, they will probably be looking for information specific to applying first. Then if they need for information that is contained there, they can continue down to the rest.

What do you think?

Comment (fureigh) [inline pages/pay-grades.md]: @rooneywp Moving those to the top sounds good to me.

**Comment by pixelwake** [comment]
@fureigh Moved the sections up. I think it's good to go if you want to take a last look and commit.

**Comment by fureigh** [comment]
@rooneywp Looks good to me! Merging.

P.S. You used the pull request perfectly. :)

**Comment by fureigh** [comment]
obama-applauds

**Pull request merged**